lib/ansible/modules/crypto/entrust/ecs_certificate.py

```diff
@@ -0,0 +1,906 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c), Entrust Datacard Corporation, 2019
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: ecs_certificate
+author:
+    - Chris Trufan (@ctrufan)
+version_added: '2.9'
+short_description: Request SSL/TLS certificates with the Entrust Certificate Services (ECS) API
+description:
+    - Create, reissue, and renew Certificates with the Entrust Certificate Services (ECS) API.
```

Comment:
```suggestion
    - Create, reissue, and renew certificates with the Entrust Certificate Services (ECS) API.
```
Or is there a reason why you capitalize "certificate" here?